components/threatconnect.js

```diff
@@ -716,13 +715,18 @@ polarity.export = PolarityComponent.extend({
               })
             );
 
+            delete `indicators.${indicatorId}.indicator__${field}PrevButtonDisabled`;
```

Comment:
Are these delete statements necessary?   Everything seems to work without them and I wasn't sure what their purpose was.